### ISSUE TITLE
Refine_module xyz.detach()

### ISSUE THREAD

**Issue by esun0087** [RosettaCommons/RoseTTAFold#89]
hi, i am confused about why xyz is detach() in Refine_module  when msa and pair is not detach()
